src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -65,6 +66,8 @@
 	public LogicProcedure transparentModelCondition;
 	public LogicProcedure isShakingCondition;
 	public LogicProcedure solidBoundingBox;
+	public NumberProcedure visualScale;
+	public NumberProcedure boundingBoxScale;

```

Comment:
If null checks in templates are a workaround for having an extra converter, I recommend assigning `new NumberProcedure(null, 1)` to the new fields here in the constructor instead of those checks.

Reply:
Null checks in procedures are because in MCreator Procedure field is allowed to have null value since procedures were introduced so this is intentional